srtcore/srt.h

```diff
@@ -571,22 +571,69 @@ enum SRT_REJECT_REASON
 // Logging API - specialization for SRT.
 
 // WARNING: This part is generated.
+
+// Logger Functional Areas
+// Note that 0 is "general".
+
+// XXX This part should better be generated
+// from logging_defs.inc.cpp, but this would require
+// that either whole srt.h is partially generated, or
+// this should be kept in a separate file.
+
+// Values 0* - general, unqualified
+// Values 1* - control
+// Values 2* - receiving
+// Values 3* - sending
+// Values 4* - management
+
+
+// Made by #define so that it's available also for C API.
+// NOTE:
 // Use ../scripts/generate-logging-defs.tcl to regenerate.
 
-#define SRT_LOGFA_GENERAL    0 // glog
 
-#define SRT_LOGFA_CONTROL    2 // mglog
-#define SRT_LOGFA_DATA       3 // dlog
+#define SRT_LOGFA_GENERAL    0 // gglog
+#define SRT_LOGFA_SOCKMGMT   1 // smlog
+#define SRT_LOGFA_CONN       2 // calog
+#define SRT_LOGFA_XTIMER     3 // xtlog
 #define SRT_LOGFA_TSBPD      4 // tslog
-#define SRT_LOGFA_REXMIT     5 // rxlog
+#define SRT_LOGFA_RSRC       5 // rslog
```

Comment:
And here as well some guide on which FA corresponds to which part of the library.

Reply:
Description might be a part of the FA definition... :thinking: